flow/designs/asap7/eth_top/config.mk

```diff
@@ -0,0 +1,50 @@
+$(info [INFO-FLOW] Ethernet Design)
+
+DESIGN_DIR                   := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
+
+export DESIGN_NAME            = eth_top
+export DESIGN_NICKNAME        = eth_top
+export DESIGN                 = eth_top
+
+export VERILOG_FILES          = $(sort $(wildcard $(abspath $(DESIGN_DIR)/../../src/$(DESIGN))/*.v))
+export SDC_FILE               = $(DESIGN_DIR)/constraint.sdc
+export ABC_AREA               = 1
+
+export CORNER                ?= BC
+
+export LIB_FILES             += $($(CORNER)_LIB_FILES)
+export LIB_DIRS              += $($(CORNER)_LIB_DIRS)
+export DB_FILES              += $($(CORNER)_DB_FILES)
+export DB_DIRS               += $($(CORNER)_DB_DIRS)
+export WRAP_LIBS             += $(WRAP_$(CORNER)_LIBS)
+export WRAP_LEFS             += $(WRAP_$(CORNER)_LEFS)
+export TEMPERATURE            = $($(CORNER)_TEMPERATURE)
+export DFF_LIB_FILE = $($(CORNER)_DFF_LIB_FILE)
+
+export ABC_CLOCK_PERIOD_IN_PS = 400
+
+export DESIGN_POWER           = VDD
+export DESIGN_GROUND          = VSS
+
+export DESIGN_DIR
+
+
+export PLATFORM               = asap7
+export DESIGN_PDK_HOME
```

Comment:
purpose?

Reply:
line 4: DESIGN_PDK_HOME              := $(realpath $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST)))))
line 34: export DESIGN_PDK_HOME
I think it can be merged in single line.
While doing script clean/ Removing unwanted variables its should be taken care @maliberty